taipy/core/_orchestrator/_dispatcher/_task_function_wrapper.py

```diff
@@ -65,7 +65,7 @@ def _write_data(self, outputs: List[DataNode], results, job_id: JobId):
                 for res, dn in zip(_results, outputs):
                     try:
                         data_node = data_manager._get(dn.id)
-                        data_node.write(res, job_id=job_id)
+                        data_node._write(res)
```

Comment:
Compare to the `DataNode.write()` method, seems like the `_DataManagerFactory._build_manager()._set()` line is missing.

Is it intended?